src/pages/community/guidelines.tsx

```diff
@@ -201,40 +190,39 @@ const CommunityGuidelines: NextPage = () => (
           order to prevent the degeneration of our chat channels.
           <ol className="list-alpha list-inside ml-6 space-y-2 py-1">
             <li>
-              This does not apply to instances where people are speaking about
-              their everyday lives or current events - only when discussion
-              devolves to the point of being solely about some political scandal
-              or controversy.
+              This does not apply to speaking about everyday lives or current events -
+              only when discussion devolves to the point of being solely about some
+              political scandal or controversy.
             </li>
           </ol>
         </li>
         <li>
           Discussing piracy or pirated software. This explicitly applies to
-          &quot;offline-mode&quot; (not authenticating users against Mojang) as
-          well.
+          &quot;offline-mode&quot; (not authenticating users against their Microsoft
+          accounts) as well.
```

Comment:
I dont know if this matters, but technically you are still authenticating against mojang, the xbl api doesnt give you info about the minecraft profile. so technically you are logging into xbl and then using that to log into your minecraft account.